### ISSUE TITLE
Error Saving Query Results as .xlsx

### ISSUE THREAD

**Issue by Kcaxtanhu** [Microsoft/sqlopsstudio#931]
- SQL Operations Studio Version: sqlops 0.26.7 (723c7cb3bfd80d87fc8a1141632089cd4808495c, 2018-02-16T21:34:47.034Z)
- OS Version: Windows_NT x64 10.0.10240
- Extensions: none
---

Issue:
My SQL Operations Studio fails to save result as Excel, but sometimes it succeeds; every time I try to save it as Excel (xlsx) it shows me a message saying that the Writer is closed or in error state.
![sqlopsstudio](https://user-images.githubusercontent.com/6532820/37601700-06a9a834-2b8b-11e8-9c95-5fbc521b3dbb.PNG)

Steps to Reproduce:

1. Execute Query
2. Click on "Save as Excel"

**Comment by kburtram** [comment]
@Kcaxtanhu I wasn't able to repro this issue.  Do you know if there's anything about the query output that be noteworthy, or other details to help isolate the repro?

**Comment by Kcaxtanhu** [comment]
@kburtram I could point the number of rows, but won't mention it, because from SQLOS I've already been able to export more rows than Excel allows. Unfortunately is something that happens time to time and for instances I've to switch to MSQL Management Studio (and add some efforts to keep the type format when exporting).

Right now I've done a test and worked, but I've updated the version to the newest!
